src/OpenLoco/src/Graphics/Gfx.cpp

```diff
@@ -81,10 +81,13 @@ namespace OpenLoco::Gfx
 
         if (header.numEntries != G1ExpectedCount::kDisc)
         {
-            Logging::warn("G1 element count doesn't match expected value:\nExpected {}; Got {}", G1ExpectedCount::kDisc, header.numEntries);
             if (header.numEntries == G1ExpectedCount::kSteam)
             {
-                Logging::info("Got Steam G1.DAT variant, will fix elements automatically.");
+                Logging::verbose("Got Steam G1.DAT variant, will fix elements automatically.");
+            }
+            else
+            {
+                Logging::warn("G1 element count doesn't match expected value:\nExpected {}; Got {}", G1ExpectedCount::kDisc, header.numEntries);
```

Comment:
This message is technically not correct now, it checked `G1ExpectedCount::kDisc` and `G1ExpectedCount::kSteam` and both did not match, could either display that we expect X or Y or drop that entirely and only log what we actually got.

Reply:
I think that we should still consider `kDisc` the standard. The Steam variant is missing some elements (translated tutorial images). But listing both `kDisc` and `kSteam` would likely just be more confusing.

Reply:
I guess, should be hardly ever happening also.